en/debug/probe_mculink.md

```diff
@@ -0,0 +1,78 @@
+# MCU-Link Debug Probe
+
+The [MCU-Link Debug Probe](https://www.nxp.com/design/design-center/software/development-software/mcuxpresso-software-and-tools-/mcu-link-debug-probe:MCU-LINK) is a cheap, fast and highly capable debug probe that can serve as a stand-alone debug and console communicator for a PX4 developer:
```

Comment:
@PetervdPerk-NXP Have you verified the operation on all the targets?

Reply:
Tested on pixhawk 5x and 6c

Reply:
6x?

Reply:
I don't have a 6X but let's assume it's the same STM32H7 it should just work